### DISCUSSION TITLE
Sites created with Cecil

### DISCUSSION THREAD

**Discussion by ArnaudLigny** [Cecilapp/Cecil#995] [Show and tell]
If you have created a site with Cecil, you can share the URL here.

- [cecil.app](https://cecil.app)
- [arnaudligny.fr](https://arnaudligny.fr)
- [cecillie.fr](https://www.cecillie.fr)
- [Jamstatic](https://jamstatic.fr)
- [libimobiledevice](https://libimobiledevice.org)
- [Thomas Ferney](https://thomas-ferney.fr)
- [Product collab'](https://www.product-collab.com)
- [Le Moulin de Prédelles](https://moulindepredelles.com)

**Comment by matthieu2607** [comment]
Hi,
And one more site: [Pixel Open](https://pixel-open.org/)
Thank you Cecil ;)

**Reply by ArnaudLigny** [reply]
Woot! \o/

**Reply by madjoutoure46-ops** [reply]
✍️✍️✍️✍️✍️✍️🥰🥰

**Reply by madjoutoure46-ops** [reply]
I'm sure they would love okay

**Comment by happytodev** [comment]
Hello Arnaud, 

I adopted Cecil, so here are my two little websites for now :

https://happyto.dev/
https://links.happyto.dev/

Thank you for your work.

**Comment by lucky1er** [comment]
Hello,

I'd like to introduce you to my new site based on Cecil : [WayPoints showcase](https://wp.close-env.com)

Thank you Cecil and all his contributors ;)

**Comment by treeandcoffee** [comment]
Hi

Here's my site: [Fokusfalter](https://fokusfalter.ch). Many thanks @ArnaudLigny for this excellent workbench!

My Pagespeed rating:
 
<img width="518" height="120" alt="mobile_pagespeed" src="https://github.com/user-attachments/assets/f03d2887-3fd7-434d-ae4e-76cb20247769" />
